synapse/tests/test_lib_net.py

```diff
@@ -364,8 +364,9 @@ def test_lib_net_listenfail(self):
         with self.getLoggerStream('synapse.lib.net', expected_msg) as stream:
 
             with s_net.Plex() as plex:
-                addr = plex.listen(('127.0.0.1', 0), None)
```

Comment:
So OSX won't allow a process to listen on port 0 (and get a randomly assigned available port)? I really don't like listening on specific ports (even randomly selected ports) in unit tests as they may already be bound (very unlikely, but it will eventually happen). I'm not sure if there's a good way around this if the above note about OSX is true.

Reply:
OK, let's leave as-is for now then